cypress/integration/pages/topicPage/index.js

```diff
@@ -18,7 +19,11 @@ Object.keys(config)
         before(() => {
           Cypress.env('currentPath', currentPath);
 
-          const newPath = `${currentPath}${overrideRendererOnTest()}`;
+          // const newPath = `${currentPath}${overrideRendererOnTest()}`;
+          const overrideSuffix = overrideRendererOnTest();
+          const newPath = `${
```

Comment:
genius

Reply:
I am just so clever this was all my idea